src/scripts/ci_build.py

```diff
@@ -201,7 +206,7 @@ def sanitize_kv(some_string):
         if target_cc in ['clang', 'gcc']:
             flags += ['--enable-sanitizers=address,undefined']
         else:
-            flags += ['--with-sanitizers']
+            flags += ['--enable-sanitizers=address']
```

Comment:
This explicitly disables the `-D_ITERATOR_DEBUG_LEVEL=1` as it doesn't seem to be compatible with ASan. Linking of the botan library fails with:

```
VCASAN.lib(vcasan.obj) : error LNK2038: mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '1' in asn1_alg_id.obj
   Creating library .\botan.lib and object .\botan.exp
.\botan.dll : fatal error LNK1319: 1 mismatches detected
```